docs/web3js-event-filters.md

```diff
@@ -0,0 +1,134 @@
+---
+id: web3js-event-filters
+title: Web3 event filters
+sidebar_label: Web3 event filters
+---
+
+## Overview
+
+The [Web3.js](https://github.com/ethereum/web3.js) library allows developers to easily listen for events from the [EVM](evm.html) 
+contained on Loom DAppChain. Also is possible to create filter for indexed values.
+
+## Filtering 
+
+Let's create a filter to get latest block created on Loom DAppChain and continuously print the block hash on console
+
+```js
+const {
+  Client, CryptoUtils, LoomProvider
+} = require('loom-js')
+const Web3 = require('web3')
+
+// Create the client
+const client = new Client(
+  'default',
+  'ws://127.0.0.1:46657/websocket',
+  'ws://127.0.0.1:9999/queryws',
+);
+
+// Create private key for first account
+const privateKey = CryptoUtils.generatePrivateKey()
+
+// Instantiate web3 client using LoomProvider as provider
+const web3 = new Web3(new LoomProvider(client, privateKey));
+
+// Create filter to get the latest block
+const filter = web3.eth.filter('latest');
+
+// Watch filter will return the hash of the latest block continuously 
+filter.watch(function (error, result) {
+  if (error) {
+    console.error(error)
+  } else {
+    console.log('Block hash', result)
+```
+
+## Filtering indexed values
+
+Another great feature is the filtering by `indexed` values, which can be used to filter specific events on contracts, for instance 
+on a contract that a specific event is only triggered if a certain `indexed` value happen.
+
+For the following contract:
+
+```js
```

Comment:
This isn't JS code, do we have a Solidity syntax highlighter?

Reply:
I don't know